python/test/unit/language/test_matmul.py

```diff
@@ -105,6 +117,8 @@ def test_simple_matmul(dtype_src_str, dtype_dst_str, BLOCK_M, BLOCK_N, BLOCK_K,
         pytest.skip("FMA matmul not supported for multiple CTAs")
     if (BLOCK_M < 64 or (BLOCK_M == 64 and BLOCK_N == 16)) and NUM_CTAS > 1:
         pytest.skip("multi-CTAs is broken for mmav2")
+    if EPILOGUE_SUBTILE and (is_hip() or NUM_CTAS > 1 or BLOCK_N >= 512):
```

Comment:
Does below change work? There is a drop in pass rate in this PR.
```suggestion
    if EPILOGUE_SUBTILE and (is_xpu() or is_hip() or NUM_CTAS > 1 or BLOCK_N >= 512):
```

Reply:
This change should in theory further worsen the pass rate, as it increases the number of cases where `pytest.skip` will be called. Most likely it should be changed to `pytest.xfail`?

Reply:
Thanks, we can continue to track the new combinations with existing issues then.